templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul>
+  <li>
+    <a href="#">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>

```

Comment:
The links currently use `href="#"`, which are placeholders. Replace these with actual URLs or use Django's `{% url %}` template tag to make the links functional.